src/field-value.ts

```diff
@@ -18,7 +18,7 @@
 
 import deepEqual from 'deep-equal';
 
-import {AnyJs} from './types';
+import {AnyDuringMigration, AnyJs} from './types';
```

Comment:
Hrm...  I'm surprised to see AnyDuringMigration exists here.  That type is usually used during automated upgrades of TypeScript to keep existing code working without rewriting it...  But we shouldn't be adding new usages of it.